packages/@webex/webex-core/test/integration/spec/services/services.js

```diff
@@ -795,8 +795,8 @@ describe('webex-core', () => {
     describe('#fetchClientRegionInfo()', () => {
       it('returns client region info', () =>
         services.fetchClientRegionInfo().then((r) => {
-          assert.isDefined(r.countryCode);
-          assert.isDefined(r.timezone);
+          assert.isDefined(r.regionCode);
```

Comment:
This test was failing due to an unrelated change in the response from the service